docs/docs/dev_docs/getting_started/updating.md

```diff
@@ -0,0 +1,100 @@
+```shell
+export SANDBOX_VERSION=0.7.7
+```
+
+Now when you run the curl command it will use the version you specified.
+To verify that it's the case check the console output of the curl command.
+You should see the following line:
+```
+Aztec Sandbox 0.7.7 is now ready for use!
+```
+
+Alternatively you can open a new terminal and use aztec-cli to get the version.
+
+#include_code node-info yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
+
+The sandbox version should be the same as the one we chose by setting the `SANDBOX_VERSION` environmental variable.
+
+## Updating Aztec CLI
+If the latest version was used when updating the sandbox then we can simply run the following command to update the CLI:
+```shell
+npm install -g @aztec/cli
+```
+
+If a specific version was set for the sandbox then we need to install the CLI with the same version:
+```shell
+npm install -g @aztec/cli@REPLACE_WITH_SANDBOX_VERSION
+```
+
+E.g.:
+```shell
+npm install -g @aztec/cli@0.7.7
+```
+
+## Updating Noir compiler
+Now we need to update the Noir compiler `nargo` to the version compatible with the sandbox.
+Use `aztec-cli` to get it:
+#include_code node-info yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
+
+Then we install the `Compatible Nargo Version` with (replace `COMPATIBLE_NARGO_VERSION` with the version from the previous command):
+```shell
+noirup -v COMPATIBLE_NARGO_VERSION
+```
+
+## Updating Noir framework
+Finally we need to update the Noir framework for Aztec contracts.
+We need to install a version compatible with our `nargo` version.
+
+To update the framework we will update a tag of the `aztec.nr` dependency in the `Nargo.toml` file to the `SANDBOX_VERSION` from above.
+Find all the dependencies pointing to the directory within `aztec.nr` framework and update the corresponding tag:
+
+```toml
+[dependencies]
+aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v<REPLACE_WITH_NEW_SANDBOX_VERSION>", directory="yarn-project/aztec-nr/aztec" }
+value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v<REPLACE_WITH_NEW_SANDBOX_VERSION>", directory="yarn-project/aztec-nr/value-note" }
+```
+
+To provide you with a concrete example we will update from version `0.7.5`:
+```toml
+[dependencies]
+aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.7.5", directory="yarn-project/aztec-nr/aztec" }
+value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.7.5", directory="yarn-project/aztec-nr/value-note" }
+```
+
+to `0.7.7`:
+
+```toml
+[dependencies]
+aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.7.7", directory="yarn-project/aztec-nr/aztec" }
+value_note = { git="https://github.com/AztecProtocol/aztec-packages", tag="aztec-packages-v0.7.7", directory="yarn-project/aztec-nr/value-note" }
+```
```

Comment:
`diff` is a valid language, so you can set it instead of toml, and prefix the changed lines with `-` and `+` to show the diff. So I'd remove the first example with the `REPLACE_WITH_NEW_SANDBOX_VERSION`, and just show a single diff snippet, which updates to the current `NoirVersion` loaded from noir-version.json.